crates/vm/levm/src/precompiles.rs

```diff
@@ -2,12 +2,16 @@ use bytes::Bytes;
 use ethrex_core::{Address, H160, U256};
 use keccak_hash::keccak256;
 use libsecp256k1::{self, Message, RecoveryId, Signature};
+use num_bigint::BigUint;
 use sha3::Digest;
 
 use crate::{
     call_frame::CallFrame,
-    errors::{InternalError, PrecompileError, VMError},
-    gas_cost::{ripemd_160 as ripemd_160_cost, sha2_256 as sha2_256_cost, ECRECOVER_COST},
+    errors::{InternalError, OutOfGasError, PrecompileError, VMError},
+    gas_cost::{
+        identity as identity_cost, modexp as modexp_cost, ripemd_160 as ripemd_160_cost,
```

Comment:
Done in [use gas_cost::precompile instead of aliasing](https://github.com/lambdaclass/ethrex/pull/1522/commits/e12d651e63e7fd58b287ba357f561519ba9c6675)